api-operator/deploy/crds/wso2.com_ratelimitings_crd.yaml

```diff
@@ -107,14 +111,18 @@ spec:
             type:
               type: string
             unitTime:
-              format: int64
               type: integer
           required:
-          - type
+          - bandwidth
+          - conditions
+          - description
+          - requestCount
+          - stopOnQuotaReach
           - timeUnit
+          - type
           - unitTime
-          - requestCount
```

Comment:
We need to check that and handle it separately. Let's only do the version upgrade here.

Reply:
Then shall we remove these required fields manually, otherwise we may unable to apply a ratelimitting CR without having these fields, because they are required.

Reply:
I have updated the CRDs with comparing to our existing CRDs, please check now